libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/BuildingExtrusionLayer.java

```diff
@@ -24,15 +24,18 @@
 import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.fillExtrusionOpacity;
 import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.visibility;
 
+/**
+ * This layer handles the creation and customization of a {@link FillExtrusionLayer}
+ * to show 3D buildings.
+ */
 public class BuildingExtrusionLayer {
 
   private static final String FILL_EXTRUSION_LAYER_ID = "extruded-buildings";
```

Comment:
This is too generic in my opinion. Could we make this name longer, and with a `mapbox` prefix/suffix so that there's a smaller likelihood of it clashing with the developer's layers?

Reply:
Switched to `private static final String MAPBOX_NAV_UI_FILL_EXTRUSION_LAYER_ID = "mapbox-nav-ui-fill-extrusion-building-layer";`